### DISCUSSION TITLE
[Vote ended on 2023-01-26] Cancel removal process for cyberark.pas from Ansible 9 (i.e. keep it in Ansible 9)

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#185] [Votes done]
https://github.com/ansible-community/community-topics/issues/180

This vote is about **cancelling the removal process of cyberark.pas from Ansible 9** (ref: https://github.com/ansible-community/community-topics/issues/168, https://github.com/ansible-community/community-topics/discussions/172). When we finished voting on starting the removal process, folks from Cyberark showed up and said that they want to start maintaining the collection again. This effectively started this month, with several new releases. See https://github.com/ansible-community/community-topics/issues/180 for more details.

Please vote +1 to cancel the removal process and **keep cyberark.pas in Ansible**. Vote -1 to not cancel the removal process; then the next Ansible release will have an announcement that it will get removed from Ansible 9 (fall 2023).

**Comment by felixfontein** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by Warkdev** [comment]
+1

**Comment by szh** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by hyclak** [comment]
+1

**Comment by acozine** [comment]
+1 and thanks to the cyberark team!

**Comment by briantist** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by oraNod** [comment]
+1

**Comment by markuman** [comment]
+1
